index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Dev-only</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.3/css/bulma.min.css">
+    <style>
+        body {
+            background: linear-gradient(-30deg, #00a4db 30%, #006cd2 70%);
+            min-height: 100vh;
```

Comment:
![Screenshot 2023-02-24 at 09-04-21 Dev-only](https://user-images.githubusercontent.com/74045243/221125353-199ee3ca-e4e0-4b07-b298-62cb83ca5fe0.png)

container height issue (I have 2160p vertical pixel)

Reply:
I updated the code with rebase, you can retry.